registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/mapper/PacketInfoMapper.java

```diff
@@ -1,505 +1,505 @@
+import io.mosip.registration.processor.packet.storage.entity.IndividualDemographicDedupeEntity;
+import io.mosip.registration.processor.packet.storage.entity.IndividualDemographicDedupePKEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegAbisRefEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegAbisRefPkEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegBioRefEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegBioRefPKEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegDemoDedupeListEntity;
+import io.mosip.registration.processor.packet.storage.entity.RegDemoDedupeListPKEntity;
+import io.mosip.registration.processor.packet.storage.exception.DateParseException;
+
+/**
+ * The Class PacketInfoMapper.
+ */
+public class PacketInfoMapper {
+
+	/** The reg proc logger. */
+	private static Logger regProcLogger = RegProcessorLogger.getLogger(PacketInfoMapper.class);
+
+	/** The Constant REGISTRATION_ID. */
+	private static final String REGISTRATION_ID = "registrationId";
+
+	/** The Constant PRE_REGISTRATION_ID. */
+	private static final String PRE_REGISTRATION_ID = "preRegistrationId";
+
+	/**
+	 * Instantiates a new packet info mapper.
+	 */
+	private PacketInfoMapper() {
+		super();
+	}
+
+	/**
+	 * Convert reg abis ref to entity.
+	 *
+	 * @param regAbisRefDto
+	 *            the reg abis ref dto
+	 * @return the reg abis ref entity
+	 */
+	public static RegAbisRefEntity convertRegAbisRefToEntity(RegAbisRefDto regAbisRefDto) {
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
+				regAbisRefDto.getReg_id(), "PacketInfoMapper::convertRegAbisRefToEntity()::entry");
+
+		RegAbisRefEntity regAbisRefEntity = new RegAbisRefEntity();
+
+		RegAbisRefPkEntity regAbisRefPkEntity = new RegAbisRefPkEntity();
+
+		regAbisRefPkEntity.setRegId(regAbisRefDto.getReg_id());
+		regAbisRefEntity.setAbisRefId(regAbisRefDto.getAbis_ref_id());
+		regAbisRefEntity.setId(regAbisRefPkEntity);
+		regAbisRefEntity.setIsActive(true);
+		regAbisRefEntity.setCrDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+		regAbisRefEntity.setUpdateDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(),
+				regAbisRefDto.getReg_id(), "PacketInfoMapper::convertRegAbisRefToEntity()::exit");
+		return regAbisRefEntity;
+	}
+
+	/**
+	 * Gets the json values.
+	 *
+	 * @param jsonNode
+	 *            the json node
+	 * @param language
+	 *            the language
+	 * @return the json values
+	 */
+	private static String getJsonValues(JsonValue[] jsonNode, String language) {
+		String value = null;
+		if (jsonNode != null) {
+			for (int i = 0; i < jsonNode.length; i++) {
+				if (jsonNode[i].getLanguage().equals(language)) {
+					value = jsonNode[i].getValue().trim();

```

Comment:
When receing data in uin generator to send to idrepo, will this code take care ?

Reply:
No,We need do separate code change in uin generator stage.
Because uin generator is taking field values from packet manager api.